packages/common/src/interfaces/currentFilter.interface.ts

```diff
@@ -16,4 +16,7 @@ export interface CurrentFilter {
 
   /** Target element selector from which the filter was triggered from. */
   targetSelector?: string;
+
+  /** If true, stringifies the search terms as is without modification */
```

Comment:
I know it's just a comment and the grammar is ok but I would prefer to see `stringify` since that is what everyone uses without thinking about the grammar ;).